.shortest.example.yml

```diff
@@ -0,0 +1,7 @@
+# This is an example config file for Shortest.
+# You can use it to configure Shortest for your repository.
+# To use it, rename it to .shortest.yml and modify it as needed.
+
+owner/repo:
```

Comment:
I think we should just have shortest.yml for this repo and it'll act like an example. Can keep comments in there to help guide how to change it.